validator/schema/upsun.json

```diff
@@ -657,35 +724,47 @@
               "properties": {
                 "spec": {
                   "type": "string",
-                  "title": "The cron schedule specification."
+                  "title": "The cron schedule specification.",
+                  "description": "The cron specification. To prevent competition for resources that might hurt performance, use `H` in definitions to indicate an unspecified but invariant time. For example, instead of using `0 * * * *` to indicate the cron job runs at the start of every hour, you can use `H * * * *` to indicate it runs every hour, but not necessarily at the start. This prevents multiple cron jobs from trying to start at the same time.\\nMore information:\\nhttps://en.wikipedia.org/wiki/Cron#Cron_expression"
                 },
                 "commands": {
                   "type": "object",
                   "properties": {
                     "start": {
                       "type": "string",
-                      "title": "The command used to start the cron job."
+                      "title": "The command used to start the cron job.",
+                      "description": "It’s run in Dash:\\nhttps://en.wikipedia.org/wiki/Almquist_shell"
```

Comment:
Maybe lets say that unless specified the shell command will run in Dash?

Reply:
```
                      "description": "By default, the command is run in Dash (till you specify something else):\\nhttps://en.wikipedia.org/wiki/Almquist_shell"
```
is this better?

Reply:
yep <3